### ISSUE TITLE
Incorrect value for getindex on multibyte unicode strings

### ISSUE THREAD

**Issue by kmsquire** [xiaodaigh/ShortStrings.jl#8]
When a multibyte unicode character is stored in a `ShortString`, `getindex` does not return that character correctly:

```julia
julia> using ShortStrings

julia> s = "∫x ∂x"
"∫x ∂x"

julia> ss = ShortString15(s)
"∫x ∂x"

julia> s[1]
'∫': Unicode U+222b (category Sm: Symbol, math)

julia> ss[1]
'â': Unicode U+00e2 (category Ll: Letter, lowercase)
```

**Comment by xiaodaigh** [comment]
Yeah, it's only meant to work with ASCII. I need to understand more about unicode before I can make any changes

**Comment by xiaodaigh** [comment]
Apologies for taking so long this has now been fixed and will be released in 0.2.4
